### PR TITLE
Generated Image work type

### LINKED ISSUE #20: Be able to create a Image object via the UI
**ACCEPTANCE CRITERIA**

- [ ] Basic metadata should work 
- [ ] The item should display 
- [ ] There  should be a thumbnail on the show page

### PR DESCRIPTION
**Pull request by little9**

### CODE REVIEW

File: app/indexers/image_indexer.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class ImageIndexer < Hyrax::WorkIndexer
+  # This indexes the default metadata. You can remove it if you want to
+  # provide your own metadata and indexing.
+  include Hyrax::IndexesBasicMetadata
+
+  # Fetch remote labels for based_near. You can remove this if you don't want
+  # this behavior
+  include Hyrax::IndexesLinkedMetadata
+
+
```
Comment (no-reply) [inline app/indexers/image_indexer.rb]: This line causes CI to fail on (I think) each of these PRs after rebase on the cop-ci branch. How do we feel about removing the line in the rebase?

File: spec/features/create_image_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      AdminSet.find_or_create_default_admin_set_id
+      login_as user
+    end
+
+    scenario do
+      visit '/dashboard'
+      click_link "Works"
+      click_link "Add new work"
+
+      # If you generate more than one work uncomment these lines
```
Comment (no-reply) [inline spec/features/create_image_spec.rb]: Do we need these lines?

Comment (no-reply) [inline spec/features/create_image_spec.rb]: Perhaps not until another model is in? But I think the rest of the PRs don't build from each other, so we might be missing something important once we merge more than one.

Comment (little9) [inline spec/features/create_image_spec.rb]: I think your right, lets discuss this first one and then use it as a guideline for the rest of them

**Comment by mark-dce** [comment]
@little9 @no-reply All of the work types will have "I can attach a file" & "I can download the attached file" type requirements.  I think it might make sense to put that type of testing on the `GenericWork` type.  This was actually part of the role that `TuftsBase` plays in the current Fedora 3 version of tufts_models - it has all the behavior (and tests) that are shared across all work types.  We could handle some of the testing side with shared examples.

Once we have the basic works in, I'll write up the additional metadata and content restriction requirements for each work type - so Images may have some specific tests about what metadata it should have and content type restrictions that should be enforced.

File: spec/features/create_image_spec.rb (hunk 2 above)
Comment (little9) [inline spec/features/create_image_spec.rb]: Whenever we do the second PR for the next work type I'll uncomment these lines

**Comment by no-reply** [comment]
@mark-dce Do we need a GenericWork for purposes other than testing? Will anyone ever create one? If no, I think we can do this with a module and some shared examples rather than a work type.

Either way, I'm happy to punt the question and trust Hyrax for the moment. I just want to make sure we get a test in for the acceptance criteria before we make any changes to the Hyrax deposit process.

File: spec/features/create_image_spec.rb (hunk 2 above)
Comment (no-reply) [inline spec/features/create_image_spec.rb]: This test doesn't seem to be running in CI, we'll want to look into that before moving forward with the other types, too.

**Pull request merged**